draft-ietf-teep-architecture.md

```diff
@@ -1177,7 +1184,7 @@ request. A TEEP Agent implementation is responsible for ensuring that it
 can recognize and decline such repeated requests. It is also responsible
 for protecting the resource usage allocated for TA management.
 
-## Compromised CA
+## Compromised or Expired CA
```

Comment:
I believe that this is slightly incorrect because a CA can get compromised but the CA cannot expire -- only a certificate can expire. 

Maybe we should write "CA Compromise or Expiry of CA Certificate"

Reply:
@hannestschofenig Done, how's it look now?